src/app/lists/ListDataview.tsx

```diff
@@ -71,7 +71,7 @@ const ListsDataView: React.FC<Props> = (props) => {
     const [errorMsg, setErrorMsg] = useState<string | null>(null);
 
     if (props.data === null) {
-        throw new Error("No data found");
+        throw new Error("We could not find any data for the lists table.");
```

Comment:
This might be a bit misleading - because an empty list is a valid data but it still means there is no data found for this list table 
The data object can only be null when there is problem fetching data from the database, so maybe be more specific like "We could not fetch the list table data at this time. Please try again later"

Reply:
So when there is a problem fetching data from the database there is already an error thrown I think? So in theory this would only be for an occasion where all the data has been removed from the lists table I think